Mathlib/LinearAlgebra/UnitaryGroup.lean

```diff
@@ -198,6 +198,23 @@ set_option linter.uppercaseLean3 false in
 
 end UnitaryGroup
 
+section specialUnitaryGroup
+
+variable (n) (α)
+
+/--`Matrix.specialUnitaryGroup` is the group of unitary `n` by `n` matrices where the determinant
+is 1. (This definition is only correct if 2 is invertible.)-/
+abbrev specialUnitaryGroup := unitaryGroup n α ⊓ MonoidHom.mker detMonoidHom
+
+variable {n} {α} {A : Matrix n n α}
```

Comment:
If you use `#where`, you can see that you already have `{A : Matrix n n α}` sitting around.
```suggestion
variable {n} {α}
```

Reply:
ohh ok, I did not know about that, interesting